src/main/antora/modules/ROOT/pages/appendix/conversions.adoc

```diff
@@ -210,6 +214,18 @@ If you require the time zone, use a type that supports it (i.e. `ZoneDateTime`)
 
 |===
 
+[[build-in.conversions.vector]]
+=== Vector type
+Spring Data has its own type for vector representation `org.springframework.data.domain.Vector`.
+While this can be used as a wrapper around a `float` or `double` array, Spring Data Neo4j supports only the `double` variant right now.
+From a user perspective, it is possible to only define the `Vector` interface on the property definition and use either `double` or `float`.
+But on the database-side those values will be persisted as `doubles` for the time being to support manual persisted arrays and
+values persisted through `setNodeVectorProperty`.
```

Comment:
Bitte so:

Neo4j will store both `double` and `float` variants as a 64bit Cypher `FLOAT` value, which is consistent with values persisted through Cypher and the dedicated `setNodeVectorProperty` function that Spring Data Neo4j uses to persist the property.